test/README.md

```diff
@@ -93,3 +93,29 @@ Run with:
 * `npm run test:integration:apk`
 * `npm run test:integration:apt`
 * `npm run test:integration:rpm`
+
+## Debugging with Tilt ##
+
+Tilt allows you to run and debug the snyk-monitor while it is running in a container. Tilt deploys the snyk-monitor using the same Helm chart that we publish to users.
+
+You can download Tilt from the [Tilt GitHub repository](https://github.com/tilt-dev/tilt#install-tilt).
+
+### Start a debugging session ###
+
+First, ensure you have the snyk-monitor namespace set up and the snyk-monitor Secret with your integration ID and dockercfg (as per the prerequisites for installing snyk-monitor).
+
+Finally, put breakpoints in the code and run `tilt up`.
```

Comment:
I really liked this tool! I didn't deep dig in the docs. Do you know if there is a way to run one of the servers in the debug mode to and on IDE look at the step by step? It would be amazing! :)

Reply:
Not sure I understand, which servers?